### PR TITLE
typo in parameter name

### PR DESCRIPTION
**Pull request by agowa**
## Description
"-ca" is not a valid parameter "-no-ca" however is and it fits the context.

## Motivation and Context
Typo in documentation

## How to test this PR?
Compare with tool source code: https://github.com/minio/certgen/blob/main/certgen.go#L40

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [x] Documentation updated
- [ ] Unit tests added/updated

### CODE REVIEW

File: docs/tls/README.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Download [`certgen`](https://github.com/minio/certgen/releases/latest) for your
 `certgen` is a simple *Go* tool to generate self-signed certificates, and provides SAN certificates with DNS and IP entries:
 
 ```sh
-./certgen -ca -host "10.10.0.3,10.10.0.4,10.10.0.5"
+./certgen -no-ca -host "10.10.0.3,10.10.0.4,10.10.0.5"
```
Comment (harshavardhana) [inline docs/tls/README.md]: IT is better to remove this flag

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: docs/tls/README.md (hunk 1 above)
Comment (agowa) [reply docs/tls/README.md]: ok

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-erasure.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :heavy_check_mark: |
| mint-fs.sh | :x: [more...](#d3eb4989-6714-4dba-80db-8be8f7567811) |
| mint-compress-encrypt-dist-erasure.sh | :x: [more...](#d2052f7a-63a8-43ea-87e6-2fc1886063a5) |
| mint-pools.sh | :x: [more...](#4c85dfa6-db10-43ac-ad27-6e2792556ec7) |
<a name="d2052f7a-63a8-43ea-87e6-2fc1886063a5"></a>
### 15555-2f79921/mint-compress-encrypt-dist-erasure.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-k8s2:31293
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0
RUN_ON_FAIL:          0

To get logs, run 'docker cp :/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 8 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 44 seconds
(4/14) Running aws-sdk-ruby tests ... done in 4 seconds
(5/14) Running awscli tests ... done in 53 seconds
(6/14) Running healthcheck tests ... done in 1 seconds
(7/14) Running mc tests ... done in 24 seconds
(8/14) Running minio-go tests ... done in 1 minutes and 16 seconds
(9/14) Running minio-java tests ... done in 35 seconds
(10/14) Running minio-js tests ... FAILED in 2 minutes and 27 seconds
{
  "name": "minio-js",
  "function": "listenBucketNotification(bucketName, prefix, suffix, events)",
  "args": "bucketName:minio-js-test-8dbf2fb7-d1cb-4b68-9e3f-dff094fdfaa1, events: s3:ObjectCreated:*",
  "duration": 128222,
  "status": "FAIL",
  "error": "Error: read ECONNRESET at TCP.onStreamRead (internal/stream_base_commons.js:209:20)"
}
(10/14) Running minio-py tests ... done in 1 minutes and 20 seconds
(11/14) Running s3cmd tests ... done in 26 seconds
(12/14) Running s3select tests ... done in 3 seconds
(13/14) Running versioning tests ... done in 3 minutes and 4 seconds

Executed 13 out of 14 tests successfully.
```
<a name="4c85dfa6-db10-43ac-ad27-6e2792556ec7"></a>
### 15555-2f79921/mint-pools.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-k8s1:30349
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0
RUN_ON_FAIL:          0

To get logs, run 'docker cp :/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 1 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 45 seconds
(4/14) Running aws-sdk-ruby tests ... done in 5 seconds
(5/14) Running awscli tests ... done in 57 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 30 seconds
(8/14) Running minio-go tests ... done in 1 minutes and 19 seconds
(9/14) Running minio-java tests ... FAILED in 2 minutes and 10 seconds
{
  "name": "minio-java",
  "function": "listObjects()",
  "args": "[bucket, prefix, recursive, 1050 objects]",
  "duration": 122006,
  "status": "FAIL",
  "error": "error occurred\nErrorResponse(code = BucketNotEmpty, message = The bucket you tried to delete is not empty, bucketName = minio-java-test-37hu4i2, objectName = null, resource = /minio-java-test-37hu4i2, requestId = 170CCD12B1E3E5F4, hostId = 644fa22b-38a8-46e4-b8cd-c29874358134)\nrequest={method=DELETE, url=http://minio-k8s1:30349/minio-java-test-37hu4i2, headers=Host: minio-k8s1:30349\nAccept-Encoding: identity\nUser-Agent: MinIO (Linux; amd64) minio-java/8.4.3\nContent-MD5: 1B2M2Y8AsgTpgAmY7PhCfg==\nx-amz-content-sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855\nx-amz-date: 20220819T165042Z\nAuthorization: AWS4-HMAC-SHA256 Credential=*REDACTED*/20220819/us-east-1/s3/aws4_request, SignedHeaders=content-md5;host;x-amz-content-sha256;x-amz-date, Signature=*REDACTED*\n}\nresponse={code=409, headers=Accept-Ranges: bytes\nContent-Length: 328\nContent-Security-Policy: block-all-mixed-content\nContent-Type: application/xml\nServer: MinIO\nStrict-Transport-Security: max-age=31536000; includeSubDomains\nVary: Origin\nVary: Accept-Encoding\nX-Amz-Request-Id: 170CCD12B1E3E5F4\nX-Content-Type-Options: nosniff\nX-Xss-Protection: 1; mode=block\nDate: Fri, 19 Aug 2022 16:50:42 GMT\n}\n >>> [io.minio.S3Base$1.onResponse(S3Base.java:690), okhttp3.internal.connection.RealCall$AsyncCall.run(RealCall.kt:519), java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128), java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628), java.base/java.lang.Thread.run(Thread.java:829)]"
}
(9/14) Running minio-js tests ... done in 1 minutes and 5 seconds
(10/14) Running minio-py tests ... done in 1 minutes and 31 seconds
(11/14) Running s3cmd tests ... done in 35 seconds
(12/14) Running s3select tests ... done in 4 seconds
(13/14) Running versioning tests ... done in 3 minutes and 5 seconds

Executed 13 out of 14 tests successfully.
```
<a name="d3eb4989-6714-4dba-80db-8be8f7567811"></a>
### 15555-2f79921/mint-fs.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-k8s7:30987
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0
RUN_ON_FAIL:          0

To get logs, run 'docker cp :/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 0 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 42 seconds
(4/14) Running aws-sdk-ruby tests ... done in 4 seconds
(5/14) Running awscli tests ... done in 52 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 20 seconds
(8/14) Running minio-go tests ... done in 1 minutes and 1 seconds
(9/14) Running minio-java tests ... done in 18 seconds
(10/14) Running minio-js tests ... FAILED in 31 minutes and 7 seconds
{
  "name": "minio-js",
  "function": "\"after all\" hook for \"removeObject(bucketName, objectName)",
  "args": "bucketName:SpecialMenùäöüexPrefix/äöüex ®©µÄÆÐÕæŒƕƩǅ 01000000 0x40 @ amȡȹɆple&0a!-",
  "duration": 7225,
  "alert": ".*'()&$@=;:+,?<>.pdf, ",
  "status": "FAIL",
  "error": "S3Error: The bucket you tried to delete is not empty at Object.parseError (node_modules/minio/dist/main/xml-parsers.js:71:11) at /mint/run/core/minio-js/node_modules/minio/dist/main/transformers.js:166:22 at DestroyableTransform._flush (node_modules/minio/dist/main/transformers.js:90:10) at DestroyableTransform.prefinish (node_modules/readable-stream/lib/_stream_transform.js:129:10) at prefinish (node_modules/readable-stream/lib/_stream_writable.js:611:14) at finishMaybe (node_modules/readable-stream/lib/_stream_writable.js:620:5) at endWritable (node_modules/readable-stream/lib/_stream_writable.js:643:3) at DestroyableTransform.Writable.end (node_modules/readable-stream/lib/_stream_writable.js:571:22) at IncomingMessage.onend (internal/streams/readable.js:670:10) at endReadableNT (internal/streams/readable.js:1333:12) at processTicksAndRejections (internal/process/task_queues.js:82:21)"
}
(10/14) Running minio-py tests ... done in 51 seconds
(11/14) Running s3cmd tests ... done in 23 seconds
(12/14) Running s3select tests ... done in 1 seconds
(13/14) Running versioning tests ... done in 1 minutes and 2 seconds

Executed 13 out of 14 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-erasure.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :heavy_check_mark: |
| mint-pools.sh | :heavy_check_mark: |
| mint-compress-encrypt-dist-erasure.sh | :x: [more...](#790d6823-2224-4485-9444-52da206156ec) |
<a name="790d6823-2224-4485-9444-52da206156ec"></a>
### 15555-718bc92/mint-compress-encrypt-dist-erasure.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-k8s6:32243
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0
RUN_ON_FAIL:          0

To get logs, run 'docker cp :/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 8 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 44 seconds
(4/14) Running aws-sdk-ruby tests ... done in 5 seconds
(5/14) Running awscli tests ... FAILED in 1 minutes and 47 seconds
{
  "name": "awscli",
  "duration": 78351,
  "function": "aws --endpoint-url http://minio-k8s6:32243 s3 presign s3://awscli-mint-test-bucket-19828/datafile-1-kB\n",
  "status": "FAIL",
  "error": "Checksum verification failed for downloaded object"
}
(5/14) Running healthcheck tests ... done in 0 seconds
(6/14) Running mc tests ... done in 26 seconds
(7/14) Running minio-go tests ... done in 1 minutes and 14 seconds
(8/14) Running minio-java tests ... done in 36 seconds
(9/14) Running minio-js tests ... done in 1 minutes and 7 seconds
(10/14) Running minio-py tests ... done in 1 minutes and 16 seconds
(11/14) Running s3cmd tests ... done in 25 seconds
(12/14) Running s3select tests ... done in 3 seconds
(13/14) Running versioning tests ... done in 3 minutes and 4 seconds

Executed 13 out of 14 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**